### PR TITLE
Fix "Mutations on a Normalized Database" example

### PR DESCRIPTION
**Pull request by fvides**
Fixed the path on merge/remove-ident* path argument, removed
superfluous :list/id first argument.

### CODE REVIEW

**Comment by fvides** [comment]
I'm studying the examples and implementing. This is a bug in this code, the PR fixes it. Thanks for the awesome job on Fulcro and documentation.

File: DevelopersGuide.adoc [hunk 1]
```diff
@@ -1708,7 +1708,7 @@ Our mutation can now become:
   [{list-id   :list/id
     person-id :person/id}]
   (action [{:keys [state]}]
-    (swap! state merge/remove-ident* [:person/id person-id] [:list/id list-id :list/people])))
```
Comment (awkay) [inline DevelopersGuide.adoc]: Your change is incorrect...the original code is correct.  Did you try this??? I would be amazed if that does anything useful.

Comment (awkay) [reply DevelopersGuide.adoc]: Thanks for trying to help, though :)   I appreciate it.

Comment (fvides) [reply DevelopersGuide.adoc]: Thank you for this great work, loving Fulcro so far :)

But there is something I don't understand about your comment:
* In application's current state, Fulcro Inspector shows a map with keys :enemies and :friends
image

* With your path, whenever I click the delete button, the item doesn't dissapear but after transacting the mutation there is some new element empty element
image

When I first saw this, after looking remote-ident* docs thought that the problem was with the route, so I deleted the first element, and the program worked just fine after the change.

What am I doing / understanding wrong?

Thanks in advance!

Comment (awkay) [reply DevelopersGuide.adoc]: The app state you are showing me is wrong. Possilby bad copy into your app.  It looks like you are missing `:ident` on the List component.  Please use Clojurians #fulcro slack for support/help.

**Comment by fvides** [comment]
Hi again.

You were right, I forgot to add :ident to PersonList. Sorry for bothering and wasting your time ':(

Once again, thanks for this great framework!

**Pull request closed without merging**